docker/local-dev/Dockerfile

```diff
@@ -35,27 +35,22 @@ RUN apt-get install -y \
         php-cli \
         php-curl \
         php-json \
-        python \
-        python-pip \
-        python-dev \
         python3 \
-        python3-pip \
         python3-dev \
+        python3-pip \
+        python3-venv \
         vim
 
 # Upgrade pip to be compatible with new platform tag formats
-RUN pip install --upgrade pip
 RUN pip3 install --upgrade pip
```

Comment:
nit: for consistency and to use latest conventions, it would be better to do `RUN python3 -m pip install --upgrade pip`. It would also be nicer to have a docker image that has one Python, and that being the right version of Python that we want, and to just call `RUN python -m ...`.

Same applies to line 53.